.taskcluster.yml

```diff
@@ -29,9 +30,13 @@ tasks:
                   $if: 'tasks_for == "github-pull-request"'
                   then: '${event.pull_request.base.repo.full_name}'
                   else:
-                      $if: 'tasks_for in ["cron", "action"]'
+                      $if: 'tasks_for == "action"'
                       # Trim git@github.com:
                       then: '${repository.url[15:-4]}'
+                      else:
+                          $if: 'tasks_for == "cron"'
+                          # Trim https://github.com/
+                          then: '${repository.url[19:]}'
```

Comment:
We use this type of construct all the time.
I wonder if it would be good to propose a `urlparse` or similar in json-e, so we might be able to use `urlparse(${repository.url})["path"]` or similar.

Reply:
I agree this is a great proposal. I'd propose we formalize the API of this `urlparse()` function in https://github.com/mozilla-releng/mozilla-repo-urls, first. Once we're happy about the data it returns, then we should be in a good position to make this behavior part of json-e[1]

[1] https://github.com/json-e/json-e/